src/node.cc

```diff
@@ -2659,9 +2668,9 @@ void FatalException(Isolate* isolate,
 
 void FatalException(Isolate* isolate, const TryCatch& try_catch) {
   HandleScope scope(isolate);
-  if (!try_catch.IsVerbose()) {
+  // if (!try_catch.IsVerbose()) {
```

Comment:
This could be supported if we cherry-pick https://codereview.chromium.org/2840803002 into libcc as a v8 patch.